server tests starter with live reload

aprooks opened the pull request:
starter for #128 

what's working: live reload of test project and tests run.

I did not use Expecto Fake helpers, not sure if they would work with 'dotnet watch'

also I noticed at least once that 2 watching process deadlocked. Not sure what to do with it atm.

Signed-off-by: Alexander Prooks <aprooks@live.ru>

forki commented:
Can't believe we are already at #129 here. ;-) this was just supposed to be a small example. But seems people are interested. Really cool tbh

aprooks commented:
* added serverTests project building on default build
* added running tests via `dotnet watch`

things to note. 

I've made 2 separate tasks for .net framework executables and for running serverTests project via dotnet
also `RunServerTests` task is put earlier than 'ClientBuild' since I prefer fail fast approach during builds and tests are being completed much faster than fable compilation

aprooks commented:
> Can't believe we are already at #129 here. ;-) this was just supposed to be a small example. But seems people are interested. Really cool tbh

small examples are very important imho :) I prefer to clone/download full working example and then adjust/delete example code instead of setting up solution from the ground every time

aprooks commented:
it is working for me now ;)

alfonsogarciacaro commented:
Nice work @aprooks! This looks good to me to merge 😄 Do you have any comment, @forki?

forki commented on test/ServerTests/Tests.fs:
```diff
@@ -0,0 +1,15 @@
+open Expecto
+open ServerCode
+
+[<Tests>]
```
Please use expecto tests without the attribute. I think we don't need that. Thx

aprooks replied:
thanks, dunno where it came from :)

aprooks replied:
@forki  removed

forki commented:
:kiss:
